lib/libsys/rfork_c18n.c

```diff
@@ -43,8 +44,10 @@
 pid_t
 rfork(int flags)
 {
-	if ((flags & RFFLAGS_SAFE) == flags)
```

Comment:
Maybe as:

```
    if (_rtld_c18n_is_enabled() && (flags & ~RFFLAGS_SAFE) != 0) {
       errno = EINVAL;
       return (-1);
    }
    return (__sys_rfork(flags));
```

Makes it clearer what the c18n-specific logic is IMO.  (Also, not sure why the entire file is #ifdef'd)